djtriggers/tests/models.py

```diff
@@ -0,0 +1,44 @@
+from time import sleep
+
+from djtriggers.exceptions import ProcessError
+from djtriggers.models import Trigger
+
+
+class NormalDummyTrigger(Trigger):
+    class Meta:
+        proxy = True
+
+    typed = 'normal_trigger_test'
+
+    def _process(self, dictionary):
+        pass
+
+
+class FailingDummyTrigger(Trigger):
+    class Meta:
+        proxy = True
+
+    typed = 'failing_trigger_test'
+
+    def _process(self, dictionary):
+        raise Exception
+
+
+class ProcessErrorDummyTrigger(Trigger):
+    class Meta:
+        proxy = True
+
+    typed = 'process_error_normal_trigger_test'
+
+    def _process(self, dictionary):
+        raise ProcessError()
+
+
+class TimelyDummyTrigger(Trigger):
```

Comment:
Timely? That means it's on time, while the code shows it is slower. Isn't that misleading?